src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $cardWidth);
+  
+  gap: 48px;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, $cardWidth);
```

no neec to repeat one column
```suggestion
    grid-template-columns: $cardWidth;
```